### DISCUSSION TITLE
is streamed on any platforms?

### DISCUSSION THREAD

**Discussion by 18rrs** [sct/overseerr#3582] [Feature Requests]
It will be verry nice if on movie/series page, we have an option that shows if the movie/series is available on streaming services. Lets say i already have Netflix or HBO and i don't want to request and download a movie that is available on  a platform that i have. 

thestreamable.com is a nice service that offer information if a title is streamed on any platforms, I bet there ar more services like this but this one i use.

**Comment by varunpan** [comment]
This is a great idea! It will prevent users from request shows/movies that they can watch on their streaming services. 
JustWatch has an API that can be used for this purpose.

**Comment by andromidasj** [comment]
Isn't this already a feature?

![CleanShot 2024-03-23 at 01 54 26@2x](https://github.com/sct/overseerr/assets/66661368/48c86bbd-545b-4d44-9281-13601cd0e031)

**Reply by Lockszmith-GH** [reply]
Came here to mention this - not clear what OP is suggesting

**Comment by AlexGuld** [comment]
Plex has that functionality. In the discover/watchlist section if a given titel is on one of the platforms you have selected it will show where to watch and even provide a direct link.

<img width="714" alt="image" src="https://github.com/user-attachments/assets/579f3173-09cb-4599-82f0-91b09a7b37ef">

So Overseerr "just" needs to take advantage of that and for instance add titels to the watchlist in plex and when it's available on a streaming service block radarr/sonarr from downloading it.

**Reply by AlexGuld** [reply]
In fact Overseerr already monitors the watchlist from plex, so it just needs the logic to also look for where the titel is available, and if so block the download and provide a direct link to that service.

**Reply by Lockszmith-GH** [reply]
But it already does that, see [this answer](https://github.com/sct/overseerr/discussions/3582#discussioncomment-8884921)

**Reply by AlexGuld** [reply]
> But it already does that, see [this answer](https://github.com/sct/overseerr/discussions/3582#discussioncomment-8884921)

Yes it shows on any service. But it would be nice if it showed if it was on one of the services you have a subscription to. Like plex does in the watchlist

**Reply by Lockszmith-GH** [reply]
ha, I see.

OP isn't really participating in this discussion, so not sure if that's what they were asking for.
But what you're describing is to manage a list of services per user in overseerr, presenting that to the user when they request.

I would rather see everywhere it exists, even if I have no subscription there.

So maybe what is needed is just a customized sort order per user - one that allows them to prioritize certain services upon others.
That, plus an option to prompt the user with an 'are you sure' dialog box presenting the list once more before hitting confirm.

**Reply by skahtee** [reply]
Would it be at all possible to add an option to "exclude streaming service" in the search filter?

 This would then allow me to exclude content from services I already subscribe to from Overseerr search results.

**Comment by crispybegs** [comment]
i've been dreaming about something like this for ages - https://www.reddit.com/r/selfhosted/comments/1ayy7eq/excludarr_extended/

"don't think this exists, but asking in case.

_[Excludarr](https://github.com/haijeploeg/excludarr) "is a CLI that interacts with Radarr and Sonarr instances. It completely manages your library in Sonarr and Radarr to only consist out of movies and series that are not present on any of the configured streaming providers. Excludarr can also re-monitor movies and series if it is not available anymore on any of the configured streaming providers. You can also configure to delete the already downloaded files of the excluded entry to keep your storage happy! "_

My wife has a habit of Overseerring stuff that's already on netflix, which is annoying.

What I'm fantasising about is a version of Excludarr that

Has a nice, simple web UI rather than CLI

Auto-runs every night / week / month / whatever, scans radarr / sonarr and makes a list of items that already appear on netflix / prime etc

Sends me that list via a slack message / email / other selectable channels so I can consider what to keep and what to delete

When I go back to the web UI the found list is there, and i can tick / untick what to keep or delete

Has a nice docker compose to do all the above

Guessing that doesn't exist, does it?"
